core/src/main/java/org/mskcc/cbio/portal/model/ExtendedMutation.java

```diff
@@ -648,30 +650,21 @@ public void setTumorSeqAllele2(String tumorSeqAllele2) {
         
         
         
-        /**
-         * Set alleles. For variant allele: one of the tumor sequence alleles
-         * which is different from the reference allele.
+    /**
+     * Set alleles. For variant allele: one of the tumor sequence alleles
+     * which is different from the reference allele.
```

Comment:
please update this part of the code documentation, since the logic has changed to something more complex. Please consider updating respective cBioPortal documentation as well, where applicable (or tooltips in mutation view, if any).